reference/masterfiles-policy-framework/cfe_internal-enterprise-federation-federation.markdown

```diff
@@ -5,4 +5,37 @@ published: true
 tags: [reference, MPF, cfe_internal, federated reporting]
 ---
 
+This policy file handles Federated Reporting setup and ongoing operations.
```

Comment:
hummm, wonder how muhc of this content we can just put into the policy as inline docs.

or, perhaps mut it into a markdown file in the federation folder and from here just suck it in using the markdown include